### PR TITLE
[507] Adds "Review this record" page

### PR DESCRIPTION
**Pull request by aliuk2012**
## Context
This is not a complete page. It adds the basic layout that will be used
and more tickets will be created to fixed and tweak this page until it's finished.

### Changes proposed in this pull request

- Review basic page mockup and flow.
- additional content and functionality will be added in another ticket.

### CODE REVIEW

**Comment by aliuk2012** [comment]
[Review Trainee Details before TRN Submission](https://trello.com/c/ipVx8Hbi/460-review-trainee-details-before-trn-submission)

**Comment by darokel** [comment]
[Review trainee details page](https://trello.com/c/5q1XU7nA/507-review-trainee-details-page)

File: spec/support/page_objects/trainees/summary.rb [hunk 1]
```diff
@@ -12,7 +12,7 @@ class Summary < PageObjects::Base
       section :training_details, PageObjects::Sections::TrainingDetails, ".training-details"
       section :degree_details, PageObjects::Sections::DegreeDetails, ".degree-details"
 
-      element :submit_for_trn_button, "input[name='commit']"
+      element :review_this_record_link, "a.govuk-button", text: "Review this record"
```
Comment (darokel) [inline spec/support/page_objects/trainees/summary.rb:15]: This should probably target by class/id. If the text changes it'll break.

**Review by darokel — COMMENTED** [review]
Left one little comment

File: spec/support/page_objects/trainees/summary.rb (hunk 1 above)
Comment (aliuk2012) [reply spec/support/page_objects/trainees/summary.rb]: 👍 I've switched to using an ID

**Review by darokel — APPROVED** [review]

**Review by defong — APPROVED** [review]

**Pull request merged**